DamageAssessment/damage_assessment.py

```diff
@@ -13,7 +13,7 @@
 BUILDING_AREA = 'gs://supporting-data2/WSF3d_v02_BuildingArea.tif'
 DDF = './damage_data/damage/DDF_Global.csv'
 MAXDAMAGE = './damage_data/damage/MaxDamage_per_m2.csv'
-COUNTRY = "Belize"
+COUNTRY = "OneDollar"
```

Comment:
Is this variable name still correct?  Seems like we should probably update the variable name to be better describe it's use

Reply:
The intention on this is to get a dollar equivalent damage.  Since some models will span country boundaries, we want to be able to spatially apply values per m^2 later in the pipeline